plugin/github.com.hashicorp.go.kms.wrapping.plugin.v2.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+option go_package = "github.com/hashicorp/go-kms-wrapping/v2/plugin;plugin";
+
+package github.com.hashicorp.go.kms.wrapping.plugin.v2;
+
+import "github.com.hashicorp.go.kms.wrapping.v2.types.proto";
+
+message TypeRequest {}
+
+message TypeResponse {
+	string type = 10;
+}
+
+message KeyIdRequest {}
+
+message KeyIdResponse {
+	string key_id = 10 [json_name="key_id"];
```

Comment:
Why is the `json_name` option needed for `key_id`?

Reply:
See https://developers.google.com/protocol-buffers/docs/proto3 -- some marshalers creating JSON from protobuf will use lowerCamelCase by default for multi-word values, so in these cases I tend to explicitly specify the desired JSON name that aligns with our normal practices.